exporters/logging-otlp/src/main/java/io/opentelemetry/exporter/logging/otlp/internal/InternalBuilder.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.logging.otlp.internal;
+
+import static java.util.Objects.requireNonNull;
+
+import io.opentelemetry.exporter.logging.otlp.OtlpJsonLoggingLogRecordExporter;
+import io.opentelemetry.exporter.logging.otlp.internal.writer.JsonWriter;
+import io.opentelemetry.exporter.logging.otlp.internal.writer.LoggerJsonWriter;
+import io.opentelemetry.exporter.logging.otlp.internal.writer.StreamJsonWriter;
+import java.io.OutputStream;
+import java.util.logging.Logger;
+
+/**
+ * Internal builder for configuring OTLP JSON logging exporters.
+ *
+ * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
+ * at any time.
+ */
+public class InternalBuilder {
+  private final Logger logger;
+  private final String type;
+  private JsonWriter jsonWriter;
```

Comment:
> InternalBuilder

it's not (primarily) for code reuse - we need it for getting the values out - at least I found no better way

https://github.com/zeitlinger/opentelemetry-java/blob/stdout-exporter-logs-core/exporters/logging-otlp/src/main/java/io/opentelemetry/exporter/logging/otlp/OtlpJsonLoggingLogRecordExporter.java#L43-L46

> passing in logger

this is to keep the same logger as before to avoid breaking changes in the output of the log line

Reply:
> InternalBuilder

I did find a way to avoid the internal builder now

however, after adding `OtlpStdoutLogRecordExporter implements LogRecordExporter`, I now need 2 builder types to produce the different exporters - but it's still easier to understand than before (I hope).